src/lib/common/globals.cpp

```diff
@@ -57,6 +57,7 @@ bool                   notifQueueStatistics = false;
 bool                   checkIdv1            = false;
 unsigned long long     inReqPayloadMaxSize  = DEFAULT_IN_REQ_PAYLOAD_MAX_SIZE;
 unsigned long long     outReqMsgMaxSize     = DEFAULT_OUT_REQ_MSG_MAX_SIZE;
+bool                   notifyOnMetadataChange = false;
```

Comment:
Note we don't want a global behaviour but a per-subscription behaviour.

In other words, `notifyOnMetadataChange` should be a field in the subscription document. Have a look to other boolean fields in subscriptions (like `onlyChangedAttrs` or `covered`) and try to do deal with them in a similar way from the point of view of subscription CRUD (Create-Read-Update-Delete).

The main difference with regards the aforementioned `onlyChangedAttrs` or `covered` is that they are within `notification` and `notifyOnMetadataChange` should be within `condition` (as it has to do with notification triggering conditions).

Reply:
Resolved